src/asciidoc/docs/platform-features/richPreview.adoc

```diff
@@ -1,143 +1,61 @@
 = Thumbnail image and custom metadata
 :page-slug: thumbnail-image
-:page-description: How to customize the thumbnail (Open Graph) image and sharing metadata for your Koji app.
+:page-description: How to customize the thumbnail (Open Graph) image and metadata for your Koji app.
 :page-banner: {imagesDir}/Koji-screenshot=1.png
 
-When a user shares a remix of your template on social media, the Koji platform renders a rich preview (Open Graph) image based on a screenshot of the Koji.
-By default, the platform takes a screenshot of the initial view.
+When you share a Koji app on social media, the Koji platform renders a thumbnail (Open Graph) image based on a screenshot of the app.
+If you were designing a web app from scratch, you would use meta tags, such as `og:image`, `og:title`, and `og:description`, to set properties for the thumbnail.
```

Comment:
Do you want me to include this graphic in the doc? If so, please point me to where it is referenced, so I can find the image.

Reply:
Nah, not this image; it was a quick mockup I did in SnagIt. If we did include something similar, it would be better to use an unfurl of a Koji app link. I suppose that's something to consider as a replacement to the existing image though. Let me see if I can come up with something.